jumpscale/sals/nginx/nginx.py

```diff
@@ -186,14 +186,32 @@ def configure(self):
         """
         self.clean()
         j.sals.fs.mkdir(self.cfg_dir)
+        username, group = self.get_user_info()
 
         configtext = j.tools.jinja2.render_template(
-            template_path=j.sals.fs.join_paths(DIR_PATH, "templates", "nginx.conf"), logs_dir=self.logs_dir,
+            template_path=j.sals.fs.join_paths(DIR_PATH, "templates", "nginx.conf"),
+            logs_dir=self.logs_dir,
+            username=username,
+            group=group,
         )
 
         j.sals.fs.write_file(self.cfg_file, configtext)
         j.sals.fs.copy_tree(f"{DIR_PATH}/resources/", self.cfg_dir)
 
+    def get_user_info(self):
+        """
```

Comment:
```
In [12]: import pwd                                                                                                                                    

In [13]: import grp                                                                                                                                    

In [14]: import os                                                                                                                                     

In [15]: pwd.getpwuid(os.getuid())                                                                                                                     
Out[15]: pwd.struct_passwd(pw_name='Jo', pw_passwd='x', pw_uid=1000, pw_gid=1000, pw_gecos='', pw_dir='/home/Jo', pw_shell='/usr/bin/zsh')

In [16]: grp.getgrgid(os.getgid())                                                                                                                     
Out[16]: grp.struct_group(gr_name='jo', gr_passwd='x', gr_gid=1000, gr_mem=[])
```

please use python for this stuff maybe should exist in a sal somewhere?

Reply:
add it under ```sals.process```?